crates/core/src/subscription/module_subscription_actor.rs

```diff
@@ -76,9 +78,13 @@ impl ModuleSubscriptionManager {
     pub async fn broadcast_event(&self, client: Option<&ClientConnectionSender>, mut event: ModuleEvent) {
         match event.status {
             EventStatus::Committed(_) => {
+                let (tx, rx) = oneshot::channel();
                 self.tx
-                    .send(Command::BroadcastCommitEvent { event })
+                    .send(Command::BroadcastCommitEvent { event, feedback_tx: tx })
                     .expect("subscription actor panicked");
+                if (rx.await).is_err() {
+                    log::error!("subscription actor not reported broadcast completion")
```

Comment:
nit: "failed to receive acknowledgement of successful evaluation from subscription actor"